swap/src/cli.rs

```diff
@@ -104,15 +103,40 @@ pub enum Command {
         )]
         monero_wallet_rpc_url: Url,
 
-        // TODO: The listen address is only relevant for Alice, but should be role independent
-        //  see: https://github.com/comit-network/xmr-btc-swap/issues/77
         #[structopt(
-            short = "a",
-            long = "p2p-address",
+            short = "l",
```

Comment:
I'd remove all the short-codes in accordance with https://github.com/comit-network/xmr-btc-swap/pull/108 which will remove them for all other things.

Reply:
Let's get #108 merged first so I can rebase once not twice :)